CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Building
+
+### Installing Jython 3
+
+You must install the `jython-dev.jar` to your local maven repository.
+
+1. Clone the Jython 3 Git repo: `git clone https://github.com/juliandolby/jython3.git`.
+1. Change directory to the cloned local Git repo: `cd jython3`.
+1. Build Jython 3: `ant`. That will produce the file `jython3/dist/jython-dev.jar`.
+1. Install the `jython-dev.jar` into your local maven repo (see [this post][SO post]):
+
+    ```bash
+    mvn install:install-file \
+	-Dfile=./jython-dev.jar \
+	-DgroupId="org.python" \
+	-DartifactId="jython3" \
+	-Dversion="0.0.1-SNAPSHOT" \
+	-Dpackaging="jar" \
+	-DgeneratePom=true
+    ```
+### Installing IDE
+
+1. Clone the [IDE repository][IDE].
+1. Change directory to `com.ibm.wala.cast.lsp`: `cd com.ibm.wala.cast.lsp`
+1. Build and install to your local Maven repo: `mvn install`
+
+### Installing WALA
+
+1. Clone the [WALA repository][WALA].
+1. Checkout the tag corresponding to the version needed, e.g., `git checkout v1.5.9`. You need to match this version with the WALA version in the `pom.xml` file found in *this* repository.
+1. Build and deploy WALA to your local maven repository. This is necessary because the test JARs are not published on Maven Central: `./gradlew publishToMavenLocal`.
+    1. If you run into [this issue](https://github.com/wala/WALA/issues/1173), try: `./gradlew publishToMavenLocal -x signRemotePublication`.
```

Comment:
Can we update these instructions in light of the updates on the corresponding WALA issue?

Reply:
Yes, but it would be different depending whether the rest JARs are released. Do we have those for 1.5.9 or does a new release need to be cut?

Reply:
I think even for 1.5.9, just running `./gradlew publishLocalPublicationToMavenLocal` should work without requiring signing; see [here](https://github.com/wala/WALA/wiki/Creating-Releases-and-Snapshots#publishing-released-versions-locally).  Does it work for you?

Reply:
Yes, but it would be ideal to have these JARs in maven local instead of building WALA locally.

Reply:
Sure, but that won't happen until after we fix the relevant issues and cut the next release (timeline unclear).  In the meantime, if we want to merge this, we should have the instructions be correct for whatever release works for this repo.

Reply:
Right. Done.

Reply:
Is there an issue on http://github.com/wala/WALA that I can track to the test JARs to be released? I'll create an issue on this slide and block on that.

Reply:
There is https://github.com/wala/WALA/issues/1201.  After that is fixed, I would have to cut a release (which should be easy).

Reply:
Oh, yes, I also forgot I had https://github.com/wala/WALA/issues/1171.